codebundles/k8s-argocd-helm-health/runbook.robot

```diff
@@ -0,0 +1,98 @@
+*** Settings ***
+Documentation       This codebundle runs a series of tasks to identify potential helm release issues related to ArgoCD managed Helm objects.
+Metadata            Author    Nitin Madhok (nmadhok@icloud.com)
+Metadata            Display Name    Kubernetes ArgoCD HelmRelease TaskSet
+Metadata            Supports    Kubernetes,AKS,EKS,GKE,OpenShift,ArgoCD
+
+Library             RW.Core
+Library             RW.CLI
+Library             RW.platform
+
+Suite Setup         Suite Initialization
+
+
+*** Tasks ***
+List all available ArgoCD Helm releases
+    [Documentation]    List all ArgoCD helm releases that are visible to the kubeconfig.
+    [Tags]    argocd    helmrelease    available    list
+    ${helmreleases}=    RW.CLI.Run Cli
+    ...    cmd=${KUBERNETES_DISTRIBUTION_BINARY} get ${RESOURCE_NAME} -n ${NAMESPACE} --context ${CONTEXT}
```

Comment:
If we leave the resource name as a variable here, then the default needs to change below. I'd also settle for hardcoding it to applications.argoproj.io

Additionally, this specific command would just list all of the applications, which is not specific to helm releases. So, when listing applications, we need to inspect the json output and and only select items that are using helm. jq or jsonpath would be appropriate here.

Reply:
@stewartshea Agreed and updated in latest commit